src/pika_rm.cc

```diff
@@ -512,6 +515,39 @@ uint32_t SyncMasterPartition::ConsensusTerm() {
 
 void SyncMasterPartition::ConsensusUpdateTerm(uint32_t term) {
   coordinator_.UpdateTerm(term);
+  CommitPreviousLogs(term);
```

Comment:
Follower invokes this interface also. However CommitPreviousLogs is leader's behavior